test/helpers/types/types.go

```diff
@@ -29,6 +30,8 @@ const (
 type ContainerLog struct {
 	ViaQCommon `json:",inline,omitempty"`
 
+	OTELApplicationLog `json:",inline,omitempty"`
```

Comment:
Do we need to add this into a different top level log type (e.g. OTELContainerLog, OTELLog)? I'm assuming OTEL format will likely be agnostic of Container versus other.  I'm thinking ahead of when we no longer normalize VIAQ at all and seems it would be more appropriate to have start the different normalization down their own path.  

Consider moving all "otel" struct to a separate file

Reply:
Move to `test/helpers/schema/otel/types.go`.